src/mono/wasm/Makefile

```diff
@@ -103,7 +105,7 @@ run-tests-%:
 	EMSDK_PATH=$(EMSDK_PATH) PATH="$(JSVU):$(PATH)" $(DOTNET) build $(TOP)/src/libraries/$*/tests/ /t:Test $(_MSBUILD_WASM_BUILD_ARGS) $(MSBUILD_ARGS)
 
 run-build-tests:
-	PATH="$(JSVU):$(PATH)" $(DOTNET) build $(TOP)/src/mono/wasm/Wasm.Build.Tests/ /t:Test $(_MSBUILD_WASM_BUILD_ARGS) $(MSBUILD_ARGS)
+	PATH="$(JSVU):$(PATH)" $(DOTNET) build $(TOP)/src/mono/wasm/Wasm.Build.Tests/ /t:Test /bl:$(LOG_PATH)/Wasm.Build.Tests.binlog $(_MSBUILD_WASM_BUILD_ARGS) $(MSBUILD_ARGS)
```

Comment:
will this work if the directory `LOG_PATH` doesn't exist?

Reply:
When could that happen? Any build would create that directory, so it should be present.

Reply:
I don't see anything in the Makefile creating `artifacts/log/$(CONFIG)`, so I just wanted to check. I assume something else is creating it :)

Reply:
TIL, apparently, msbuild does that! I just tried it out.